ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3184,3 +3184,27 @@ def test092_select_large_number_of_test_units_in_one_testplan(self):
                 for testunit in testunit_names:
                     self.assertIn(testunit, result['test_units'])
 
+    def test093_add_dynamic_field_in_order_section1(self):
+        """
+        order : create new order: check that dynamic field added in order section 1 will be displayed
+        in create new order screen
+        LIMS-7870
+        """
+        section1_fields=[]
+        self.order_page.add_dynamic_text_field_to_section1()
```

Comment:
@yara-hossam you can replace this by adding feild using api

Reply:
updated